src/ir_Bosch.cpp

```diff
@@ -106,6 +106,15 @@ void IRBosch144AC::setTemp(const uint8_t degrees) {
   setTempRaw(kBosch144TempMap[temp - kBosch144TempMin]);
 }
 
+/// Set the temperature. Allow 0.5 degree steps.
+/// @param[in] degrees The temperature in degrees celsius.
+void IRBosch144AC::setTemp(const float degrees) {
```

Comment:
It would be best to have just one version of `setTemp()` that takes a `float` as an arguement.
i.e. Do all the conversion/processing in just one method/procedure.